test.py

```diff
@@ -1279,8 +1279,9 @@ def parse_cmd_line() -> argparse.Namespace:
                 suites. Each name is used as a substring to look for in the
                 path to test file, e.g. "mem" will run all tests that have
                 "mem" in their name in all suites, "boost/mem" will only enable
-                tests starting with "mem" in "boost" suite. Default: run all
-                tests in all suites.""",
+                tests starting with "mem" in "boost" suite, and
+                "boost/memtable_test::test_hash_is_cached" to narrow down to
+                a certain test. Default: run all tests in all suites.""",
```

Comment:
So far the terminology was "suite/test::case", so the correct text should be "... to narrow down to a certain test case"

Reply:
sure. i actually struggle a little bit between test versus test case. =)